### ISSUE TITLE
[1.12.2] [Bug] Certain textures contain permanently shaded areas.

### ISSUE THREAD

**Issue by ExtraBodybuilder** [Shadows-of-Fire/Plants#120]
https://imgur.com/a/kb4kPd5

These photos use incense cedar stairs as an example. I don't have a complete list of objects because I'm playing survival, nor am I 100% sure it's a Plants issue. However, I felt I'd report it here just in case it _is._ Can confirm it also happens on Brazillian pine stairs.

When holding a torch _directly in _front__ of the stairs, the shaded area disappears. Torches placed in a wall in front of the stairs, behind the stairs, underneath the stairs, or any other variant of stair-block-shenanigans have no effect.

It's not a game-shattering bug, so don't feel pressure or anything. Thanks for all your hard work, lemme know if this is WAD and/or there's something obvious I've missed. Peace.

**Comment by Shadows-of-Fire** [comment]
#114 
Not sure why it happens, not sure if it's plants or if it's smth else.  It shouldn't be Plants, because my code is lthe same as vanilla stairs, but I don't know of anything that could cause it.

**Comment by ExtraBodybuilder** [comment]
Yeah, I figured that was probably the case. I've tried disabling every other mod I have/regenning and it still happens, but I can't see anything wrong in your code. I noticed that one shader seemed to fix the issue, but I'm not a big fan of the shaders I've found, so - not exactly a fix, especially if it's a problem nobody else seems to have.

The weirdest thing? I'm new to minecraft. My pal hooked me up with my starter mods, same setup as on their rig, and it seems to all work fine there. Anyway, I'll close this if you like; just thought I'd let you know in case it ever becomes apparent...

But like you said, I'm not thinking it's you. Anyhow, thanks for the response.

**Comment by ExtraBodybuilder** [comment]
**SOLVED IT.**

Closing the issue and including the solution, in case someone else has a similar problem. I only just got a computer more recent than, say, the last ten years or so. I'd had options -> video settings -> smooth lighting set to **OFF.**

For whatever reason, setting that option to **minimum** seems to have allowed for everything to display correctly. Anyhow, thanks for the help. Hope this feedback ends up being useful in some way. Ahaha! 

Since I'm closing up, I'll just say that my favourites are blue flax, hellebore, and the ashen and black kauri trees. Thanks for all your hard work!
